BLL/Data/Repositories/DrinkRepository.cs

```diff
@@ -18,7 +18,12 @@ public DrinkRepository(DrinkAppContext context)
 
         public IEnumerable<Drink> GetAllDrinks()
         {
-            return _context.Drinks.AsEnumerable();
+            return _context.Drinks.AsEnumerable().OrderBy(x => x.Name);
+        }
+
+        public IQueryable<Drink> GetAllDrinksAsQueryable()
+        {
+            return _context.Drinks.AsQueryable();
```

Comment:
As we talked yesterday, I've added OrderBy, so Drinks are now ordered by Name. Additionally I've added GetAllDrinksAsQueryable to implement it in SearchByAlcoholContent